src/App.css

```diff
@@ -1,8 +1,26 @@
+@import 'react-toastify/dist/ReactToastify.css';
+
 #root {
   max-width: 100%;
-  
-  margin: 0 auto;
+  box-sizing: border-box;
+  margin: 0 ;
   padding: 0;
-  text-align: center;
 }
 
+
+.login-grid{
+  display: flex;
+  justify-content: space-evenly;
+}
+
+a{
+  text-decoration: none;
+  color:  rgba(21, 44, 91, 1);
```

Comment:
We should stick to MUI' design system from colors, spacing and so on.
This will make our styles easier to update.

Using hardcoded colors like this will make us need to handle light and dark colors for them manually. On the other hand, if we stick to the MUI colors, the theme colors will change automatically without needing to exert any effort in managing the colors.